packages/bsky/src/api/app/bsky/notification/putActivitySubscription.ts

```diff
@@ -10,39 +11,60 @@ export default function (server: Server, ctx: AppContext) {
     auth: ctx.authVerifier.standard,
     handler: async ({ input, auth }) => {
       const actorDid = auth.credentials.iss
-      const {
-        subject,
-        key: existingKey,
-        activitySubscription: { post, reply },
-      } = input.body
-
+      const { subject, activitySubscription } = input.body
       if (actorDid === subject) {
         throw new InvalidRequestError('Cannot subscribe to own activity')
       }
 
-      const payload: SubjectActivitySubscription = {
-        subject,
-        activitySubscription: { post, reply },
-      }
-      const key = existingKey ?? TID.nextStr()
+      const existingKey = await getExistingKey(ctx, actorDid, subject)
+      const enabled = isActivitySubscriptionEnabled(activitySubscription)
+
       const stashInput = {
         actorDid,
         namespace: NamespaceAppBskyNotificationDefsSubjectActivitySubscription,
-        payload,
-        key,
+        payload: <SubjectActivitySubscription>{
```

Comment:
This is a type cast similar to using the `as` keyword.  It's a little uncommon so just making sure that was intended.